dtfs-central-api/src/types/db-models/utilisation-data.d.ts

```diff
@@ -0,0 +1,35 @@
+import { WithId } from 'mongodb';
+
+export type UtilisationData = WithId<{
+  facilityId: string;
+  /**
+   * The '_id' of the associated report from the 'utilisationReports' collection
+   */
+  reportId: string;
+  /**
+   * The 'id' (not '_id') of the associated bank in the 'banks' collection
+   */
+  bankId: string;
+  /**
+   * 1-indexed month of the start of the report period
+   * example: 6 (for June)
+   */
+  month: number;
+  /**
+   * year of the start of the report period
+   * example: 2023
+   */
+  year: number;
+  exporter: string;
+  baseCurrency: string;
+  facilityUtilisation: number;
+  totalFeesAccruedForTheMonth: number;
+  monthlyFeesPaidToUkef: number;
+  paymentCurrency: string;
```

Comment:
@ChristianMcc - I wasn't confident enough on my understanding of these values to add a description, so could use your expertise here.